File: 896. Monotonic Array/Solution.java
class Solution {
    public boolean isMonotonic(int[] nums) {
        if(nums[0]<nums[nums.length-1]){
            for(int i = 1; i< nums.length;i++){
                if(nums[i-1]>nums[i]) return false;
            }
        }else{
            for(int i = 1; i< nums.length;i++){
                if(nums[i-1]<nums[i]) return false;
            }
        }

        return true;
    }
}

File: 896. Monotonic Array/Solution.py
class Solution:
    def isMonotonic(self, nums: List[int]) -> bool:
        if nums[0]<nums[-1] :
            for i in range(1,len(nums)):
                if nums[i-1]> nums[i]:
                    return False
        else :
            for i in range(1,len(nums)):
                if(nums[i-1]< nums[i]):
                    return False
        
        return True